pkgs/tools/networking/vpnc/default.nix

```diff
@@ -20,22 +24,22 @@ stdenv.mkDerivation {
   # `ifconfig' as found in net-tools (not GNU Inetutils).
   propagatedBuildInputs = [ nettools ];
 
-  nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [libgcrypt perl openssl ];
+  nativeBuildInputs = [ makeWrapper ]
+    ++ lib.optional (!opensslSupport) pkg-config;
+  buildInputs = [ libgcrypt perl ]
+    ++ (if opensslSupport then [ openssl ] else [ gnutls ]);
 
   makeFlags = [
     "PREFIX=$(out)"
     "ETCDIR=$(out)/etc/vpnc"
     "SCRIPT_PATH=$(out)/etc/vpnc/vpnc-script"
-  ];
+  ] ++ lib.optional opensslSupport "OPENSSL_GPL_VIOLATION=yes";
 
   postPatch = ''
     patchShebangs makeman.pl
   '';
 
   preConfigure = ''
-    sed -i 's|^#OPENSSL|OPENSSL|g' Makefile
-
     substituteInPlace "vpnc-script" \
       --replace "which" "type -P" \
       --replace "awk" "${gawk}/bin/awk" \
```

Comment:
This entire section fits more into postPatch.

Reply:
I dunno — this is mostly about setting paths to runtime dependencies, which is what the configure phase is all about.  To me, the patch phase is more about making changes that aren't just setting paths to dependencies.

But I don't think it matters very much, especially because I'd like to replace this with the maintained fork at some point in the future, which doesn't require any of these changes (I think).

So, I don't mind much either way — if you still think it should be `postPatch`, I'll change it toto `postPatch`.